content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -93,6 +96,42 @@ Pods. This means that an ill-behaved Pod that floods the API server with
 requests cannot prevent leader election or actions by the built-in controllers
 from succeeding.
 
+### Request Width
+
+The above description of concurrency management is the baseline story.
+In it, all requests have equal "width": each takes up one _unit of
```

Comment:
As a non-native speaker, metaphors and informal terms are very difficult to understand.
Can we try avoid using words such as "width", "seats", "unit of concurrency"?

Reply:
I get it on width and seat, but “unit of concurrency” should be neutral between cultures.

Reply:
Some similar term instead of “unit of concurrency” would be fine as well.

Reply:
I have revised to eliminate usage of the word "width".

I kept "seat" -- it already appears many times in the document --- but reworded to clearly define the relevant meaning rather than expect the reader to bring any intuition.